app/settings.php

```diff
@@ -15,21 +15,22 @@
           [
             'cache' => [
               'enabled' => PHP_SAPI !== 'cli',
-              'redis' => "redis://${_ENV['REDIS_USERNAME']}:${_ENV['REDIS_PASSWORD']}@${_ENV['REDIS_HOST']}:${_ENV['REDIS_PORT']}"
+              'redis' => "redis://{$_ENV['REDIS_USERNAME']}:{$_ENV['REDIS_PASSWORD']}@{$_ENV['REDIS_HOST']}:{$_ENV['REDIS_PORT']}"
             ],
             'db' => [
-              'dsn' => "pgsql://${_ENV['POSTGRES_USER']}:${_ENV['POSTGRES_PASSWORD']}@${_ENV['POSTGRES_HOST']}:${_ENV['POSTGRES_PORT']}/${_ENV['POSTGRES_DB']}"
+              'dsn' => "pgsql://{$_ENV['POSTGRES_USER']}:{$_ENV['POSTGRES_PASSWORD']}@{$_ENV['POSTGRES_HOST']}:{$_ENV['POSTGRES_PORT']}/{$_ENV['POSTGRES_DB']}"
             ],
             'queue' => [
-              'dsn' => "amqp://${_ENV['AMQP_USER']}:${_ENV['AMQP_PASS']}@${_ENV['AMQP_HOST']}:${_ENV['AMQP_PORT']}"
+              'dsn'      => "amqp://{$_ENV['AMQP_USER']}:{$_ENV['AMQP_PASS']}@{$_ENV['AMQP_HOST']}:{$_ENV['AMQP_PORT']}",
+              'prefetch' => 100
             ],
-            'displayErrorDetails' => (isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'development'),
+            'displayErrorDetails' => (isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'dev'),
```

Comment:
@okiedork should this be kept as "development"?

Reply:
Here we use `dev`.